docs/en/ingest-management/overview.asciidoc

```diff
@@ -24,7 +24,7 @@ To learn about installation options, refer to <<elastic-agent-installation>>.
 [[unified-integrations]]
 == {integrations}
 
-Our https://www.elastic.co/integrations/[integrations] provide an easy way to connect Elastic to external services and systems, and quickly get insights or take action.
+{integrations-docs}[Elastic integrations] provide an easy way to connect Elastic to external services and systems, and quickly get insights or take action.
```

Comment:
I'm not a fan of ever telling users that something is easy...but I'll leave this as-is because we should probably revisit this overview at some point to improve it.